grails-app/views/layouts/react.gsp

```diff
@@ -3,6 +3,13 @@
 <head>
     <title><g:layoutTitle default="OpenBoxes" /></title>
     <link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.min.css">
+    <g:if test="${session.useDebugLocale}">
```

Comment:
Please either use `main.gsp` or add **all** header scripts. It is missing `hotjar`, `googleSiteTag` and `NewRelic`

Reply:
I've decided to migrate those from `main.gsp` to `react.gsp` layout, since our `main.gsp` is different in Grails 3 and in Grails 1, so if I wanted to use the `main.gsp`, the page looked like this:
![Screenshot from 2023-09-12 17-23-29](https://github.com/openboxes/openboxes/assets/93163821/306bd1ad-e47b-4e07-a062-49162f1727f0)